README.rst

```diff
@@ -10,11 +10,11 @@ Why Should I Use This?
 ----------------------
 
 The biggest reason to use twine is that it securely authenticates you to PyPI
-over HTTPS using a verified connection while ``python setup.py upload`` `uses
-HTTP and exposes your credentials <http://bugs.python.org/issue12226>`_.  This
-means anytime you use it you expose your username and password to being
-sniffed. Twine uses only verified TLS to upload to PyPI protecting your
-credentials from theft.
+over HTTPS using a verified connection while ``python setup.py upload`` `only
+recently stopped using HTTP <http://bugs.python.org/issue12226>`_.  This means
```

Comment:
>"Only recently" is deceptive and either needs to be expanded

I thought the `Versions` field of the linked bugreport would be informative enough while not distracting the user from twine's rationale. I have updated the whole paragraph though.

>Further, not using HTTP simply isn't enough to make distutils secure.

I didn't write that anywhere. Even the commit message doesn't, although I tried hard to keep that line within 50 chars.